openedx/adg/lms/tests/test_helpers.py

```diff
@@ -0,0 +1,25 @@
+""" Tests for LMS helper """
+import pytest
+
+from common.djangoapps.student.tests.factories import UserFactory, UserProfileFactory
+from openedx.adg.lms.helpers import get_user_first_name
+
+from .constants import FIRST_NAME, FIRST_PART_OF_FULL_NAME
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'first_name, full_name, expected_first_name', [
+        (FIRST_NAME, '', FIRST_NAME),
+        ('', f'{FIRST_PART_OF_FULL_NAME} last_part_of_full_name', FIRST_PART_OF_FULL_NAME)
```

Comment:
Please fix the pylint issue:

`openedx/adg/lms/tests/test_helpers.py:14:13: E0001: Format strings are only supported in Python 3.6 and greater (<unknown>, line 14) (syntax-error)`

Reply:
The test if also failing because of that.

Reply:
But in KOA we are on python 3.8 I guess we need to fix our pipelines.

Reply:
Creating a followup ticket for that.